src/turnkeyml/common/filesystem.py

```diff
@@ -317,6 +317,8 @@ class Keys:
     PARAMETERS = "parameters"
     # Histogram of ONNX operators used in the model
     ONNX_OPS_COUNTER = "onnx_ops_counter"
+    # Number of FLOPs in the model
+    ONNX_FLOPS_COUNTER = "onnx_flops_counter"
```

Comment:
Can we add a few lines about this feature to the tool user guide?

Reply:
@ramkrishna2910 is there _any_ documentation of ONNX stats in the tool user guide? You/we might need to start a section documenting the prior stats before Phil has a place to document this one.